src/middlewared/middlewared/plugins/failover_/event.py

```
@@ -733,7 +733,12 @@ def vrrp_backup(self, job, fobj, ifname, event):
         # do something with iscsi service here
 
         logger.info('Syncing encryption keys from MASTER node (if any)')
-        self.run_call('failover.call_remote', 'failover.sync_keys_to_remote_node')
+        try:
+            self.run_call('failover.call_remote', 'failover.sync_keys_to_remote_node')
+        except Exception as e:
+            ignore = (errno.ECONNRESET, errno.ECONNREFUSED, errno.ECONNABORTED, errno.EHOSTDOWN)
+            if isinstance(e, CallError) and e.errno not in ignore:
```

Do we silently ignore everything that is not `CallError`?